src/components/dialogs/resubmit.vue

```diff
@@ -0,0 +1,252 @@
+<template>
```

Comment:
Although I implemented this, this feature is not fully designed yet (as discussed at Sprint Planning today).

In theory, the API is not specifying the REAPPLY (resubmit) action, so the button will not display and the user should not end up on this dialog atm.